README.md

```diff
@@ -84,17 +84,82 @@ The primary supported way of running PeopleMath in a staging or production confi
 
 ### Authentication & Authorization
```

Comment:
We need to reorganise this a bit, as this content doesn't all belong in the App Engine section. It should be possible for people to use Firebase Auth using the code you've built, without running on App Engine.

Don't worry about this for now though - I'll work on this later.